packages/sockethub/src/js/client.js

```diff
@@ -86,7 +70,9 @@
         } else if (event === 'disconnect') {
           this.online = false;
         }
-        this.__handlers[event](obj);
+        for (let cb of this.__handlers[event]) {
```

Comment:
Looks to me like this should be `const` instead of `let`?

```suggestion
        for (const cb of this.__handlers[event]) {
```